### ISSUE TITLE
[BUG] The governance for establishing a working group is unclear

### ISSUE THREAD

**Issue by ericearl** [bids-standard/bids-website#375]
### Describe your problem in detail.

The governance for working groups is currently missing information, and as such is unclear. Current governance:

> A working/interest group can be established under the approval of the BIDS Steering Group. This is typically but not limited to being formed for the purpose of advancing the BIDS community, not the standard. Each group will appoint 1-2 chairs. Members of these groups can have cross appointments in other groups (e.g., BEP Working Group). These groups do not necessarily dissolve after some duration or event, unless stated in their proposal.
>
> The working/interest group formation is formalized through an open letter via a “read-only” Google Document addressed to the BIDS Steering Group. The open letter will be posted on: the [BIDS-Specification GitHub repository](https://github.com/bids-standard/bids-specification), [Google Group](https://groups.google.com/forum/#!forum/bids-discussion), and [social media channels](https://github.com/bids-standard/bids-specification?tab=readme-ov-file#BIDS-communication-channels). This proposal will state what their group aims and goals are.

### Describe what you expected.

I think resolving the following three points would be able to make better governance here.

1. There should be a way to message the steering group without using individual email addresses. Maybe a [bids-steering@gmail.com](mailto:bids-steering@gmail.com) should be made and monitored? That or make the governance state to open an issue here and tag @bids-standard/steering.
2. It is not clear from the governance where on the BIDS Specification an open letter should be posted on the repository. As a PR? As an issue?
3. A working group member can send a message to the Google Group with the open letter, but whose responsibility is it to post the formation of a working group notice to BIDS social media channels? The working group or the BIDS maintainers?

**Comment by sappelhoff** [comment]
Thanks for pointing out these points @ericearl! Here is my opinion:

> There should be a way to message the steering group without using individual email addresses. Maybe a [bids-steering@gmail.com](mailto:bids-steering@gmail.com) should be made and monitored? That or make the governance state to open an issue here and tag bids-standard/steering.

I think we should ask work groups to message `bids.maintenance+steering@gmail.com` and then the BIDS maintainers (who have access to the `bids.maintenance` Gmail account) could add a forwarding rule to the steering group members. We use this "forwarding syntax" already for a number of things, such as Code of Conduct complaints (see this section: https://bids.neuroimaging.io/governance.html#e-help). the BIDS maintainers would then update the forwarding rule once a year, when steering group members are changed.

> It is not clear from the governance where on the BIDS Specification an open letter should be posted on the repository. As a PR? As an issue?

I would say it should be an email to the bids-discussion list.

> A working group member can send a message to the Google Group with the open letter, but whose responsibility is it to post the formation of a working group notice to BIDS social media channels? The working group or the BIDS maintainers?

I think it should be the BIDS maintainers responsibility to further distribute the open letter to the social media and other communication channels. The working group leads should provide suitable text snippets for that, so that the maintainers don't have to formulate a lot.

**Comment by ericearl** [comment]
@sappelhoff I don't think we have any differing opinions, so I am fully in favor of yours which all make sense here. So can I also send my working group open letter there now? I will send my open letter to the bids-discussion list very soon. I can understand a short blurb for social media channels, but could those maybe just be re-used for "other communication channels"? I'm not sure which ones fall into the "other" category.

**Comment by sappelhoff** [comment]
> so can I also send my working group open letter there now? 

+1

> but could those maybe just be re-used for "other communication channels"?

+1

> I'm not sure which ones fall into the "other" category.

I mean all channels listed here: https://github.com/bids-standard/bids-specification/?tab=readme-ov-file#bids-communication-channels

but I don't think it'd be necessary to really post on every single communication channel 🤔 Perhaps the "main" channels should be mandatory, and the "social media" channels optional?

I suggest: 

1. full letter on bids-discussion (this is the main/official post)
2. small posts on all other "main" channels, referring to 1.
3. optional posts on "social media" channels, referring to 1.
